cmd/snap/complete.go

```diff
@@ -28,9 +31,39 @@ func (s installedSnapName) Complete(match string) []flags.Completion {
 	return ret
 }
 
+func completeFromSortedFile(fn, match string) ([]flags.Completion, error) {
+	fd, err := os.Open(fn)
```

Comment:
s/fd/file/ or similar

Let's please not get used to calling file objects or readers/writers as descriptors. A file descriptor is a number, obtained in this case via file.Fd()